src/components/PlanterQuote.js

```diff
@@ -0,0 +1,104 @@
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+
+import quoteImgReverse from '../images/quote-reverse.svg';
+import quoteImg from '../images/quote-symbol.svg';
+import DataTag from './common/DataTag';
+
+function PlanterQuote({
+  quote,
+  name,
+  photo,
+  initialDate,
+  location,
+  reverse = false,
+}) {
+  const theme = useTheme();
+  return (
+    <Box
+      sx={{
+        background: theme.palette.background.OrangeGreenGradientDark,
+        boxSizing: 'border-box',
+        height: 'fit-content',
+        justifyContent: 'center',
+        padding: 8,
+        borderRadius: { xs: 2, sm: 0 },
+        px: { xs: 12 },
+        py: { sm: 10 },
+        mt: { xs: 30, sm: 0 },
```

Comment:
![2022-01-10 17 54 22 localhost 2f340872a478](https://user-images.githubusercontent.com/59486980/148868845-9e271871-e74f-4975-a3c6-1364fadad974.png)
It's hard to determine exactly how much space is between these components in the area I marked. This spacing is from the margin being set here. Ideally, spacing between sibling components should be controlled by the parent component that is implementing those components. This practice makes page layouts easier to maintain and improves component reusability.

In this case however, because the component layout is a bit complex and the component is not currently being reused I am ok with leaving this as-is.

Reply:
If there is the need to change the mobile layout later we can always open a new issue :) The organization page is still a WIP.